antiplag/tests/test_views.py

```diff
@@ -31,24 +31,28 @@ def test_handle_duplicates(self):
             document=self.document2,
             matched_docs=[
                 {"name": "test_name", "elastic_id": 123, "percentage": 0.19},
-                {"name": "test_name", "id": result1.id, "percentage": 0.19},
+                {"name": "test_name", "id": str(self.document.id), "percentage": 0.19},
```

Comment:
I don't think this is correct from functional and logic perspective. Previously, the matched docs contained the id from result1, and now you changed it into document?

Reply:
Looks like previous tests were not correct. `matched_docs` should have IDs of documents after all, not IDs of results.

Reply:
Got it. Good job for finding it.